src/components/atom/button/styles.scss

```diff
@@ -0,0 +1,65 @@
+.button {
+  display: flex;
+  gap: 8px;
+  align-items: center;
+  justify-content: center;
+  border-radius: 100px;
+  border: 1px solid transparent;
+  padding: 16px 24px;
+  color: white;
+  font-weight: 400;
+  cursor: pointer;
+
+  &-primary {
+    @extend .button;
+    background-color: #e6007a !important;
+
+    &:hover {
+      background-color: black !important;
+    }
+
+    &-large {
+      @extend .button-primary;
+      height: 48px;
+      width: 195px;
+      font-size: 13px;
+    }
+
+    &-small {
+      @extend .button-primary;
+      height: 48px;
+      width: 189px;
+      font-size: 11px;
+    }
+  }
+
+  &-secondary {
+    @extend .button;
+    background-color: white !important;
+    color: black;
+    border: 1px solid black;
+    height: 46px;
+    width: 165px;
+    font-weight: 400;
+    font-size: 11px;
+
+    &:hover {
+      background-color: #e6007a !important;
+      color: white;
+      border: none;
+    }
+  }
+
+  &-interactive {
+    @extend .button-primary;
+    width: 424px;
```

Comment:
Please use proportional width/height instead of hard coded values. In this case, you can put padding for height.